ui/apps/platform/src/Containers/Vulnerabilities/components/FilterAutocomplete.tsx

```diff
@@ -48,6 +50,7 @@ function FilterAutocompleteSelect({
     searchFilter,
     onFilterChange,
     searchOptions,
+    isRegexMatcher,
```

Comment:
Apropos my earlier comment about making this change backwards-compatible,

```suggestion
    isRegexMatcher = false,
```